db_access.py

```diff
@@ -1,99 +1,99 @@
 import sqlite3
 import datetime, hashlib
 from flask import app, g
-
 from chirpr import app
 
+
+def db_exec(*args, commit=False):
+    conn = get_db()
+    if commit == True:
+        conn.execute(*args)
+        conn.commit()
+    else:
+        return conn.execute(*args)
+
+
 def hash_ps(text):
     return hashlib.sha224(text.encode('utf-8')).hexdigest()
 
 
+def add_chirp(text, uID, MD=False):
+    if MD == False or '<script' in text:
+        text = text.replace('<', '&lt;') # to stop script tags
```

Comment:
http://flask.pocoo.org/docs/0.11/security/

Make sure you are solving a real problem. It might be that Flask has already prevented this sort of XSS by use of Jinja2 and `{{ }}`